ntbs-service/Pages/Shared/_ManualTestResultTable.cshtml

```diff
@@ -52,7 +52,12 @@
                         @result?.SampleType?.Description
                     </nhs-table-item>
                     <nhs-table-item heading-text="@Html.DisplayNameFor(_ => header.Result)">
-                        @result.ResultDisplayString
+                        @result.Result.GetDisplayName()
+                        @if (!string.IsNullOrEmpty(result.NoResultReason))
+                        {
+                            <br/>
```

Comment:
I'm a bit surprised that Razor didn't encode the newline as a `<br>` tag.

If we were doing this more widely, I would suggest a css based solution like the one suggested [here](https://stackoverflow.com/questions/4220381/replace-line-break-characters-with-br-in-asp-net-mvc-razor-view) but as a one-off I think this is ok